### PR TITLE
docs: document client idle timeout

### LINKED ISSUE #3107: docs: Idle Timeout in ClientTrafficPolicy
*Description*:
Document idle timeout in ClientTrafficPolicy.

relates to #2861

### PR DESCRIPTION
**Pull request by yaelSchechter**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #3107

### CODE REVIEW

File: site/content/en/latest/tasks/traffic/client-traffic-policy.md [hunk 1]
```diff
@@ -573,5 +573,69 @@ curl -v http://$GATEWAY_HOST/get \
 request timeout
 ```
 
+### Configure Client HTTP Idle Timeout
```
Comment (shawnh2) [inline site/content/en/latest/tasks/traffic/client-traffic-policy.md:576]:
```suggestion
### Configure Client HTTP Idle Timeout

```

File: site/content/en/latest/tasks/traffic/client-traffic-policy.md [hunk 2]
```diff
@@ -573,5 +573,69 @@ curl -v http://$GATEWAY_HOST/get \
...
+The idle timeout is defined as the period in which there are no active requests. When the idle timeout is reached the connection will be closed.
+For more details see [here](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/protocol.proto#envoy-v3-api-field-config-core-v3-httpprotocoloptions-idle-timeout:~:text=...%7D%0A%7D-,idle_timeout,-(Duration)%20The)
```
Comment (shawnh2) [inline site/content/en/latest/tasks/traffic/client-traffic-policy.md:578]:
```suggestion
For more details see [here](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/protocol.proto#envoy-v3-api-field-config-core-v3-httpprotocoloptions-idle-timeout:~:text=...%7D%0A%7D-,idle_timeout,-(Duration)%20The).
```

File: site/content/en/v1.0.1/tasks/traffic/client-traffic-policy.md [hunk 3]
```diff
@@ -464,5 +464,69 @@ curl -v http://$GATEWAY_HOST/get \
 request timeout
 ```
 
+### Configure Client HTTP Idle Timeout
```
Comment (shawnh2) [inline site/content/en/v1.0.1/tasks/traffic/client-traffic-policy.md:467]: ditto

File: site/content/en/latest/tasks/traffic/client-traffic-policy.md (hunk 1 above)
Comment (yaelSchechter) [reply site/content/en/latest/tasks/traffic/client-traffic-policy.md:576]: Thanks, fixed all the comments

**Review by arkodg — APPROVED** [review]
LGTM thanks !

**Pull request merged**